### DISCUSSION TITLE
in internal  it raise cannot find package when  i exec  'rye add  xx' ?

### DISCUSSION THREAD

**Discussion by beincy** [astral-sh/rye#569] [Q&A]
I havd already  add the config to my pyproject.toml
```
[[tool.rye.sources]]
name = "default"
url = "http://172.16.2.13/pypi/simple/"
```
 After i executed the command `rye add flask`   I got this error `error: did not find package 'flask'`.
But if I rewrite the file, add the package information  to dependencies section
```
dependencies = [
    "flask"
]
```
, and then execute the command 'rye sync', it will execute successfully.
What should I do?

**Comment by mitsuhiko** [comment]
Today using the `add` command uses `unearth` under the hood. This makes me wonder if unearth is not able to work with your index. What software are you using to manage and serve up this index?
